org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/ZkReportViewer.java

```diff
@@ -1217,6 +1243,14 @@ else if (event.getName().equals(ON_RENDER_REPORT_EVENT))
 	        	this.onCtrlKeyEvent(keyEvent);
         	}
 		}
+        else if (event.getTarget() instanceof ProcessModalDialog)
+        {
+        	if (!DialogEvents.ON_WINDOW_CLOSE.equals(event.getName())){
+        		return;
+        	}
+        	
+        	hideBusyMask();

```

Comment:
if this is means to run hideBusyMask() if event is on_window_close, it is more readable to use a positive condition block instead, i.e if (DialogEvents.ON_WINDOW_CLOSE.equals(event.getName())) hideBusyMask();